index.html

```diff
@@ -1,13 +1,102 @@
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
+                width="24"
+                height="24"
+                viewBox="0 0 24 24"
+              >
+                <path
+                  stroke="currentColor"
+                  stroke-linecap="round"
+                  stroke-linejoin="round"
+                  stroke-width="2"
+                  d="M6 18v-3m4 3v-6m4 6V9m4 9V6"
+                />
+              </svg>
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
+                width="24"
+                height="24"
+                viewBox="0 0 24 24"
+              >
+                <path
+                  d="m1 9l2 2c4.97-4.97 13.03-4.97 18 0l2-2C16.93 2.93 7.08 2.93 1 9zm8 8l3 3l3-3a4.237 4.237 0 0 0-6 0zm-4-4l2 2a7.074 7.074 0 0 1 10 0l2-2C15.14 9.14 8.87 9.14 5 13z"
+                />
+              </svg>
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
+                width="24"
+                height="24"
+                viewBox="0 0 24 24"
+              >
+                <path
+                  d="M3.12 17.6A3.12 3.12 0 0 1 0 14.48V8.72A3.12 3.12 0 0 1 3.12 5.6h16.011a3.12 3.12 0 0 1 3.12 3.12v5.76a3.12 3.12 0 0 1-3.12 3.12zM1.108 8.72v5.76a2.012 2.012 0 0 0 2.011 2.01h16.012a2.011 2.011 0 0 0 2.009-2.01V8.72a2.012 2.012 0 0 0-2.009-2.01H3.119c-1.11 0-2.01.9-2.011 2.01zm2.679 6.4a1.312 1.312 0 0 1-1.307-1.308V9.385a1.31 1.31 0 0 1 1.305-1.31h14.678a1.31 1.31 0 0 1 1.311 1.31v4.427a1.316 1.316 0 0 1-1.311 1.308zm18.87-5.41a2 2 0 0 1 .014 3.773l-.014.004z"
+                />
+              </svg>
+            </section>
+          </section>
+        </head>
+
+        <nav class="navigation_bar">
+          <h1>My Logo</h1>
+          <svg
+            xmlns="http://www.w3.org/2000/svg"
+            width="24"
+            height="24"
+            viewBox="0 0 24 24"
+            class="box"
+          >
+            <path
+              d="M20 11H4c-.6 0-1 .4-1 1s.4 1 1 1h16c.6 0 1-.4 1-1s-.4-1-1-1zM4 8h16c.6 0 1-.4 1-1s-.4-1-1-1H4c-.6 0-1 .4-1 1s.4 1 1 1zm16 8H4c-.6 0-1 .4-1 1s.4 1 1 1h16c.6 0 1-.4 1-1s-.4-1-1-1z"
+            />
+          </svg>
+        </nav>
+      </header>
+
+      <section class="content">
+        <section class="details">
+          <h1>Hey There.  <br>
+             I’m Matembe.</h1>
+          <h3>I'm a Software Developer</h3>
+          <p>
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hesitate to contact me.
+          </p>
+
+          
+        
+
+        <section class="icons">
+          <img src="img/Vector.png" alt="" />
+          <img src="img/Icon - Linkedin.png" alt="" />
+          <img src="img/peace.png" alt="" />
+          <img src="img/twitter.png" alt="" />
+          <img src="img/m.png" alt="" />
+        </section>
```

Comment:
Kindly update your list of social links to be wrapped in a `<ul>` tag instead of a `<section>` tag. This is for semantic reasons as ul tags are used to wrap most lists in a webpage

- In addition to that, please wrap each img in a `<li>` tag and make sure that the images are all links using an `<a>` tag. Your final code should look similar to something like this
`<ul>`
     `<li><a><img/></a></li>`
     `<li><a><img/></a></li>`
     `<li><a><img/></a></li>`
`</ul>`